modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm

```diff
@@ -1,15 +1,15 @@
 /// Maximum an Hemophage will drain, they will drain less if they hit their cap.
-#define HEMOPHAGE_DRAIN_AMOUNT 50
+#define HEMOPHAGE_DRAIN_AMOUNT 150 //Hemophages are now more lethal when they drain living people. This changes the charge of unwilling feeding to assault by proxy, instead of battery - Bubberstation change
```

Comment:
People might use this for RP or other such things, why nerf them so significantly by increasing this to immediately dangerous levels?

Reply:
Also, can this be used offensively as a weapon?